### PR TITLE
ASL ISLR datasets

### PR DESCRIPTION
**Pull request by J22Melody**

### CODE REVIEW

File: sign_language_datasets/datasets/asl_citizen/asl_citizen.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  journal={arXiv preprint arXiv:2304.05934},
+  year={2023}
+}
+"""
+
+_DOWNLOAD_URL = 'https://download.microsoft.com/download/b/8/8/b88c0bae-e6c1-43e1-8726-98cf5af36ca4/ASL_Citizen.zip'
+
+
+_POSE_URLS = {
+    "holistic": "/shares/volk.cl.uzh/zifjia/ASL_Citizen/poses",
```
Comment (AmitMY) [inline sign_language_datasets/datasets/asl_citizen/asl_citizen.py:40]: hosted on gs://sign-language-datasets/poses/holistic/ASLCitizen.zip

File: sign_language_datasets/datasets/asl_citizen/asl_citizen.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                if self.builder_config.include_pose is not None:
+                    if self.builder_config.include_pose == "holistic":
+                        mediapipe_path = path.join(_POSE_URLS['holistic'], row[1].replace('.mp4', '.pose'))
+
+                        if path.exists(mediapipe_path):
+                            with open(mediapipe_path, "rb") as f:
+                                pose = Pose.read(f.read())
+                                datum["pose"] = pose
+
+                                # TODO: remove debug code
```
Comment (AmitMY) [inline sign_language_datasets/datasets/asl_citizen/asl_citizen.py:121]: remove i guess

**Review by AmitMY — CHANGES_REQUESTED** [review]
lgtm, just change to hosted poses

**Comment by J22Melody** [comment]
@AmitMY could you merge this PR?

**Comment by AmitMY** [comment]
merging without review.

**Pull request merged**